mobile/build.gradle

```diff
@@ -20,37 +21,39 @@ repositories {
 
 android {
     compileSdkVersion 23
-    buildToolsVersion "22.0.1"
-
+    buildToolsVersion '25'
     defaultConfig {
         applicationId "com.dexdrip.stephenblack.nightwatch"
-        minSdkVersion 15
+        minSdkVersion 20
```

Comment:
ok - so it is not going to be that easy. Updating the code to the AppCompatActivity places a bunch of library dependancies that is breaking the runtime. I am looking at it now but it is not something as easy as changing a number